Rendering/Core/vtkLabeledContourMapper.cxx

```diff
@@ -603,6 +603,7 @@ bool vtkLabeledContourMapper::PrepareRender(vtkRenderer *ren, vtkActor *act)
       continue;
       }
     metric.Value = scalars->GetComponent(ids[0], 0);
+    metric.Value = std::fabs(metric.Value) > 1e-6 ? metric.Value : 0.0;
```

Comment:
@sankhesh should there also be an added test (and rounding) for negative values?